indico/modules/events/registration/models/registrations.py

```diff
@@ -53,6 +53,29 @@ def _get_next_friendly_id(context):
     return increment_and_get(Event._last_friendly_registration_id, Event.id == event_id)
 
 
+registrations_tags_table = db.Table(
+    'registrations_registration_tags',
+    db.metadata,
+    db.Column(
+        'registration_id',
+        db.Integer,
+        db.ForeignKey('event_registration.registrations.id', ondelete='CASCADE'),
```

Comment:
When using `ondelete='CASCADE'` you need `passive_deletes=True` on the corresponding relationships (or backrefs)